ui/helpers/utils/optimism/fetchEstimatedL1Fee.js

```diff
@@ -19,14 +19,20 @@ const OPTIMISM_GAS_PRICE_ORACLE_ABI = [
 const OPTIMISM_GAS_PRICE_ORACLE_ADDRESS =
   '0x420000000000000000000000000000000000000F';
 
-export default async function fetchEstimatedL1Fee(txMeta, ethersProvider) {
+export default async function fetchEstimatedL1Fee(
+  chainId,
+  txMeta,
+  ethersProvider,
+) {
+  const networkId = Number(chainId);
```

Comment:
chainId is passed in as '0x1a4', while `Web3Provider` is expecting `type Networkish = Network | string | number;`